configs/application/config.json

```diff
@@ -80,10 +80,10 @@
 			"undockDisbandsEntireGroup": false,
 			"fillHolesOnUndock": true,
 			"tabbing": {
-				"enabled": false
+				"enabled": true
```

Comment:
Tabbing and tiling should be disabled by default. Please double check with Sean to make sure

Reply:
Fixed.